iOS/Flows/Home/_New/new_MessageCell.swift

```diff
@@ -29,12 +43,54 @@ class new_MessageCell: UICollectionViewCell {
     override func layoutSubviews() {
         super.layoutSubviews()
 
-        self.replyCountLabel.sizeToFit()
+        self.messageContainerView.expandToSuperviewWidth()
+        self.messageContainerView.expand(.bottom, padding: 20)
+        self.messageContainerView.roundCorners()
+
+        self.messageTextView.expandToSuperviewWidth()
+        self.messageTextView.sizeToFit()
+        self.messageTextView.centerOnXAndY()
+
+        self.repliesContainerView.expandToSuperviewWidth()
+        self.repliesContainerView.top = 10
+        self.repliesContainerView.expand(.bottom)
+        self.repliesContainerView.roundCorners()
+
+        self.replyCountLabel.setSize(withWidth: self.repliesContainerView.width)
         self.replyCountLabel.pin(.right)
         self.replyCountLabel.pin(.top)
 
-        self.textView.width = self.contentView.width
-        self.textView.sizeToFit()
-        self.textView.centerOnXAndY()
+        self.repliesTextView.expandToSuperviewWidth()
+        self.repliesTextView.sizeToFit()
+        self.repliesTextView.centerOnXAndY()
+    }
+
+    func setMessage(_ text: String) {
+        self.messageTextView.text = text
+
+        self.setNeedsLayout()
+    }
+
+    func setReplies(_ replies: [String]) {
```

Comment:
I would imagine this would eventually be Messagable array?

Reply:
Yeah that's a good idea.